newsfragments/103.feature

```diff
@@ -0,0 +1,10 @@
+**Features**
+
```

Comment:
The Features section be added by `towncrier`, so it must not be here. Also, towncrier will transform each newsfragment into a bullet list item, and append a link to the related PR/issue. So this should really be a simple paragraph, and if there is more to say, it should go into the documentation.